acme.sh

```diff
@@ -520,13 +520,13 @@ _get() {
   if _exists "curl" ; then
     if [ "$onlyheader" ] ; then
       $CURL -I -A "User-Agent: $USER_AGENT" $url
-      if [[ $? -gt 0 ]] ;then
-       _err "Curl failed with exit code $?, trying running script in DEBUG=2 mode"
+      if [ $? -gt 0 ] ;then
```

Comment:
In the '_get()' function,  please move the change to the end of the function, so it covers both the curl and wget.  There is a variable `ret` for use.

And I think just using equal sign is ok,  instead of  `-gt`